easybuild/easyconfigs/l/libxc/libxc-3.0.0-intel-2017b.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxc'
+version = '3.0.0'
+
+homepage = 'http://www.tddft.org/programs/octopus/wiki/index.php/Libxc'
+description = """Libxc is a library of exchange-correlation functionals for density-functional theory.
+ The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+# Results for some functionals (e.g. mgga_c_tpss) deviate with too aggressive optimization settings.
+toolchainopts = {'lowopt': True, 'optarch': False}
```

Comment:
@migueldiascosta Isn't enabling `lowopt` sufficient for avoiding problems? See #4199

Reply:
@boegel not on Haswell

```
FAIL: xc-run_testsuite
                                             Unpolarized    Polarized 
 * hyb_mgga_x_ms2h                H           FAIL          OK
 * mgga_x_br89                    BrOH        FAIL         FAIL
 * mgga_x_br89                    BrOH+       FAIL         FAIL
 * mgga_x_ms1                     H            OK          FAIL
 * mgga_x_ms2                     H           FAIL          OK
```

Reply:
@migueldiascosta OK, then we better try to make that consistent in all `libxc` easyconfigs (and also try to enable tests everywhere)?

Reply:
@boegel I'd think so. Of course, true reproducibility is a mirage, and it may be that the tests are too strict (but I believe that's no longer the case here), so if there is something in particular with avx2 (I suppose that's the difference) that's messing up the results, I'd disable it and let people enable it if any performance difference is worth it and they have the numerical difference under control

Reply:
#5256